lib/linguist/languages.yml

```diff
@@ -4461,6 +4461,24 @@ SRecode Template:
   extensions:
   - ".srt"
   language_id: 335
+SSH Config:
+  type: data
+  group: INI
+  filenames:
+  - "ssh-config"
+  - "ssh_config"
+  - "sshconfig"
+  - "sshd-config"
+  - "sshd_config"
+  extensions:
+  - ".ssh_config"
+  - ".ssh-config"
```

Comment:
The `.ssh_config` and `.ssh-config` extensions have only 22 and 8 files on GitHub. I think we should remove them from this pull request.